### ISSUE TITLE
Prisma db push errors when running generate with `unreachable`

### ISSUE THREAD

**Issue by DanielMoln** [prisma/prisma#19130]
Hi Prisma Team! The following command just crashed.

## Command

`db push`

## Versions
      
| Name        | Version            |
|-------------|--------------------|
| Platform    | windows            | 
| Node        | v16.17.0           | 
| Prisma CLI  | 4.10.1             | 
| Engine      | aead147aa326ccb985dcfed5b065b4fdabd44b19| 

## Error
```
Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Datasource "db": MongoDB database "dev" at "duncansbydev.l51xedi.mongodb.net"
Applying the following changes:

[+] Collection `Gallery`
[+] Collection `TextTranslations`
[+] Collection `Dog`
[+] Unique index `TextTranslations_postId_key` on ({"postId":1})
[+] Unique index `Dog_pedigreeName_key` on ({"pedigreeName":1})


Your database indexes are now in sync with your Prisma schema. Done in 1.48s

Running generate... (Use --skip-generate to skip the generators)
Oops, an unexpected error occurred!
unreachable

Please help us improve Prisma by submitting an error report.
Error reports never contain personal or other sensitive information.
Learn more: https://pris.ly/d/telemetry

√ Submit error report » Yes
Submitting...
Oops. We could not send the error report.
Error report submission failed due to:  FetchError: invalid json response body at https://error-reports.prisma.sh/ reason: Unexpected token H in JSON at position 0
    at D:\Web Projects\DuncansbySpottedDalmataKennel\server\node_modules\prisma\build\index.js:91151:36
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async request (D:\Web Projects\DuncansbySpottedDalmataKennel\server\node_modules\prisma\build\index.js:95529:10)
    at async createErrorReport (D:\Web Projects\DuncansbySpottedDalmataKennel\server\node_modules\prisma\build\index.js:95504:18)     
    at async sendPanic (D:\Web Projects\DuncansbySpottedDalmataKennel\server\node_modules\prisma\build\index.js:95614:23)
    at async panicDialog (D:\Web Projects\DuncansbySpottedDalmataKennel\server\node_modules\prisma\build\index.js:95858:24)
    at async handlePanic (D:\Web Projects\DuncansbySpottedDalmataKennel\server\node_modules\prisma\build\index.js:95819:3) {
  type: 'invalid-json'
}
```

**Comment by Jolg42** [comment]
Note: 
The issue about the endpoint error is https://github.com/prisma/prisma/issues/17891

These 2 issues also have the same `unreachable` error for `generate`
https://github.com/prisma/prisma/issues/18762
https://github.com/prisma/prisma/issues/17889

**Comment by DanielMoln** [comment]
I removed prisma from package.json, then I installed it again. Now, it works.

**Comment by Jolg42** [comment]
Thanks for the update though this is quite weird 🤔 

@DanielMoln Do you mean removing the Prisma CLI and / or the Prisma Client package? By the way, which package manager and version are you using?

**Comment by DanielMoln** [comment]
Yes, I removed them. I do not sure at the moment, but I used the x.10.x version and now, I am using x.13.x versions and everything fine.

**Comment by Jolg42** [comment]
Thanks! I'll close as "potentially fixed since" until we get a reproduction on 4.13 or newer
